chapters/profile-linkage.md

```diff
@@ -0,0 +1,478 @@
+# N Linkage Profile
+
+The Linkage Profile describes relationships between components and sub-components that make up a deployment. One can envision the Linkage Profile as a textual representation of a directed graph where the vertices are individual documents describing one software and the edges are the relationships between the components described in the two vertices. This representation conveys information about the deployment's state and evolution through its supply chain.
```

Comment:
Perhaps a better description here is "relationships between components and where to get more information about the connected components"